tags.go

```diff
@@ -186,3 +173,19 @@ func (tags Tags) ToFastTags() *FastTags {
 	}
 	return &r
 }
+
+func (tags Tags) ToMap(fieldMaps... map[string]interface{}) map[string]interface{} {
+	fields := map[string]interface{}{}
+	for _, fieldMap := range fieldMaps {
+		for k, v := range fieldMap {
+			fields[k] = v
+		}
+	}
+	if tags != nil {
+		tags.Each(func(k string, v interface{}) bool {
```

Comment:
1. The same problem with priorities.
2. You use `Each` while you know that it's just a map here. Why?
3. You completely repeat the code while it could be implemented as a common function for both methods.

Fixes for p.2 and p.3 are mutually exclusive, choose :)

Reply:
1. fixed
2&3. I chose the second option )